test/integration/integration_test.go

```diff
@@ -68,8 +69,30 @@ func TestMain(m *testing.M) {
 	runner := util.MinishiftRunner{CommandPath: minishiftBinary}
 	if runner.IsCDK() {
 		godogTags += "~minishift-only"
+		isoName = "rhel"
+		fmt.Println("Test run using CDK binary with RHEL iso.")
 	} else {
 		godogTags += "~cdk-only"
+		switch os.Getenv("MINISHIFT_ISO_URL") {
+		case "b2d":
+			fmt.Println("Test run using Boot2Docker iso image.")
+			isoName = "b2d"
+		case "centos":
+			fmt.Println("Test run using CentOS iso image.")
+			isoName = "centos"
+		case "":
+			fmt.Println("Test run using Boot2Docker iso image.")
+			isoName = "b2d"
+		default:
+			envIso := os.Getenv("INTEGRATION_ISO")
```

Comment:
@hferentschik +1 for regexp check, did not realized that. It seems like enough coverage, one can use her own iso to test, however it is required at least include b2d/boot2docker or centos in filename. Seems fair to me.